include/HubFramework/HUBViewModel.h

```diff
@@ -20,6 +20,7 @@
  */
 
 #import "HUBSerializable.h"
+#import <UIKit/UIKit.h>
```

Comment:
Do we need to import UIKit here, how about a forward class declaration of `UINavigationItem`?